docs/PhDInterns.md

```diff
@@ -0,0 +1,29 @@
+---
+hide:
+  - navigation
+---
+
+# PhD Intern Scheme
+
+Our internships are aimed at current PhD students looking for an industrial placement of around five months with the right to work in the UK. The projects are focussed on innovation, in particular around getting the most value out of NHS data.
+
+The projects often have a focus on emerging data science techniques and so we advertise mainly to data science programmes, however previous interns have come from other disciplines such as clinical, mathematics, computer science and bioinformatics, which have added huge value through the range of approaches and knowledge.
+
+For more information and details on how to apply see the [Scheme Overview](https://nhsx.github.io/nhsx-internship-projects/overview.html) page on the microsite
+
+For details on open projects see the [Projects](https://nhsx.github.io/nhsx-internship-projects/projects.html) page on the microsite
+
+Available outputs from previous projects can also be seen at [Previous Projects](https://nhsx.github.io/nhsx-internship-projects/completed.html) on the microsite
+
+### Current Projects 
+Currently our interns are working on the following projects in two waves.  These are the original briefs they applied to and their work and outputs will be available on our organisation GitHub.
+
+
+
+|**Wave 6** | February - July 2024 |
+|---|---|
+|![area](https://img.shields.io/badge/NLP-yellow) &nbsp; ![technique](https://img.shields.io/badge/Curation-lightgrey) &nbsp; ![datamodality](https://img.shields.io/badge/MultiModal-blueviolet)|[NHS Language Corpus Extension](https://nhsx.github.io/nhsx-internship-projects/nhs-language-corpus/)|
+|![area](https://img.shields.io/badge/Explainability-orange) &nbsp; ![technique](https://img.shields.io/badge/Fairness-lightgrey) &nbsp; ![datamodality](https://img.shields.io/badge/MultiModal-blueviolet) | [Understanding Fairness and Explainability in Multi-modal Approaches within Healthcare](https://nhsx.github.io/nhsx-internship-projects/advances-modalities-explainability/)|
```

Comment:
also, we dont have these projects in the projects page, is it worth transferring them over, with some detail on progress and linking to that?

Reply:
Personally would prefer to keep them separate for clarity.